fubarhouse/ansible-role-youtrack
# Issue 7

* update travis tests to use docker
* add redhat docker images to tests
* reformat defaults file
* remove webserver tasks
* include handler tasks (commons handlers for vhosts)
* segregate os-specific tasks
* ensure apt-cache task doesn't run on non-debian/ubuntu systems
* switch out apt task for package task
* change list of dependencies
* remove vhost files
* no longer uses the .jar file provided by jetbrains, now uses the zip in the same standard as the other roles associated to jetbrains
* working configurable port
* replace init.d files
* remove script controlling service for init.d replacements
* adds service on redhat
* rewrite test playbook
